client-templates/container-registry-agent/accept.json.sample

```diff
@@ -0,0 +1,48 @@
+{
+  "public": [
+  {
+    "method": "POST",
+      "path": "/api/v1/import/app"
+  },
+  {
+    "method": "POST",
+    "path": "/api/v2/import/done"
+  },
+  {
+    "method": "POST",
+    "path": "/api/v2/import/result"
+  }
+  ],
+  "private": [
+  {
+    "method": "GET",
+    "path": "/healthcheck",
+    "origin": "${CR_AGENT_URL}"
+  },
+  {
+    "method": "GET",
+    "path": "/validate-credentials",
+    "origin": "${CR_AGENT_URL}"
+  },
+  {
+    "method": "GET",
+    "path": "/get-state-identifier",
+    "origin": "${CR_AGENT_URL}"
+  },
+  {
+    "method": "GET",
+    "path": "/list*",
```

Comment:
Looking at other broker client examples, I think you can split this into two entries, one with the path: "/list" (for listing repos) and another one with the path "/list/:repo" (for listing tags).

Reply:
Updated